python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -498,83 +499,131 @@ async def shutdown(self):
 
         self.log.info("Disabling ATAOS corrections")
 
-        try:
-
-            await self.rem.ataos.cmd_disableCorrection.set_start(
-                disableAll=True, timeout=self.long_timeout
-            )
-        except Exception:
-            self.log.exception("Failed to disable ATAOS corrections. Continuing...")
+        if check.ataos:
+            self.log.debug(f"Disabling ATAOS corrections.")
```

Comment:
I find it confusing that a boolean called `check.ataos` which evaluates to `true` leads to disabling corrections. Shouldn't this boolean be `false` instead?

Reply:
That is because this is a "shutdown" method... the `check` attribute is a way to skip certain components from general operations in case they are not available or something like that. I should probably document this somewhere.

Reply:
Ah, sorry, I missed that. Thanks for the explanation.